scripts/tools/annotation/annotation.py

```diff
@@ -11,9 +11,9 @@
 from sklearn.linear_model import LogisticRegression
 from joblib import dump, load
 
-dataset_path = '/home/amercier/code/20bn-realtimenet/fitness_tl_benchmark/'
+dataset_path = '/home/twentybn/Code/sense/dataset/'
```

Comment:
should not commit this change nor the next one

Reply:
ok